javagony/OddOrEvenChecker.java

```diff
@@ -0,0 +1,12 @@
+public class OddOrEvenChecker {
+    public static void main(String[] args) {
+        int number = Integer.parseInt(args[0]);
+        try{
+            int remainder = 1/(number%2);
+            System.out.printf("%d is odd\n", number);
+        }
+        catch(ArithmeticException e){
+            System.out.printf("%d is odd\n", number);
+        }
```

Comment:
This change makes sure that the user has passed a command-line argument that can be parsed to an `int`.
```suggestion
        try{
            int number = Integer.parseInt(args[0]);
            int remainder = 1/(number%2);
            System.out.printf("%d is odd\n", number);
        } catch(ArrayIndexOutOfBoundsException | NumberFormatException e) {
            System.out.println("Please pass a number in the command-line");
            return;
        } catch(ArithmeticException e){
            System.out.printf("%d is odd\n", number);
        }
```